src/migration/1651572315442-init.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class init1651572315442 implements MigrationInterface {
```

Comment:
Will TypeORM continue to work if we capitalise these class names?

Reply:
It in fact does, apparently the command to generate these don't capitalize it by default for some odd reason. Have updated it now